docs/src/tutorials/complete_simulation_chain_IVC_lit.jl

```diff
@@ -90,6 +90,11 @@ is_depleted(sim_undep.point_types)
 println("Depleted:   ", get_active_volume(sim.point_types))
 println("Undepleted: ", get_active_volume(sim_undep.point_types));
 
+# The depletion voltage can also be estimated from the simulation with the depleted detector.
+# For this, the user has to choose a contact to which to apply the voltage, as well as a range to probe:
+
+get_depletion_voltage(sim, 2, 1700:0.5:2000)
```

Comment:
This is by the way also something that we do in the tutorial (with contact id: 2)